### PR TITLE
Animation delay examples

### PR DESCRIPTION
**Pull request by Kadif-Ravik**
### Description

Examples of the animation delay property

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Kadif-Ravik** [comment]
@SSwiniarski

File: content/css/concepts/animations/terms/animation-delay/animation-delay.md [hunk 1]
```diff
@@ -87,3 +87,7 @@ div {
   }
 }
 ```
+
+![Examples 1 animations(https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example1.gif)
```
Comment (dakshdeepHERE) [inline content/css/concepts/animations/terms/animation-delay/animation-delay.md:91]:
```suggestion
![Example 1 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example1.gif)
```

File: content/css/concepts/animations/terms/animation-delay/animation-delay.md [hunk 2]
```diff
@@ -87,3 +87,7 @@ div {
...
+
+![Examples 2 animations(https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example2.gif)
```
Comment (dakshdeepHERE) [inline content/css/concepts/animations/terms/animation-delay/animation-delay.md:93]:
```suggestion
![Example 2 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example2.gif)
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Have 2 comments for you

File: content/css/concepts/animations/terms/animation-delay/animation-delay.md [hunk 3]
```diff
@@ -87,3 +87,7 @@ div {
   }
 }
 ```
+
+![Example 1 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example1.gif)
+
+![Example 2 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example2.gif)
```
Comment (dakshdeepHERE) [inline content/css/concepts/animations/terms/animation-delay/animation-delay.md:93]: @Kadif-Ravik can be specific in the tags what the animation will do. Like Animations example with delay of 2 seconds.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Have one last request for you to work on @Kadif-Ravik

File: content/css/concepts/animations/terms/animation-delay/animation-delay.md [hunk 4]
```diff
@@ -87,3 +87,7 @@ div {
   }
 }
 ```
+
+![Example 1, delay animation for 2 seconds](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example1.gif)
+
+![Example 2, after the first animation delay second animation for 6000 milliseconds](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example2.gif)
```
Comment (dakshdeepHERE) [inline content/css/concepts/animations/terms/animation-delay/animation-delay.md:93]:
```suggestion
![Example 2, after the first animation delay the second animation for 6000 milliseconds](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example2.gif)
```

**Review by dakshdeepHERE — APPROVED** [review]

**Comment by dakshdeepHERE** [comment]
Looks good for second review @Kadif-Ravik 🚀

**Review by SSwiniarski — APPROVED** [review]
@Kadif-Ravik Looks good!

**Pull request merged**